ui/pages/espace-pro/form/index.tsx

```diff
@@ -46,13 +30,13 @@ export default function FormCreatePage() {
 
         const response = (await apiPost("/appointment-request/context/create", {
           body: { idCleMinistereEducatif: cleMinistereEducatif, referrer },
-        })) as any // TODO not any
+        })) as IAppointmentRequestContextCreateResponseSchema
```

Comment:
```suggestion
        }))
```
On peut supprimer le `as ...` si le typage de la réponse de la route est correct